unitex/src/fr/umlv/unitex/frames/UnitexFrame.java

```diff
@@ -170,6 +130,7 @@ public void textFrameOpened(boolean taggedText) {
 						convertFst.setEnabled(true);
 						exportTfstAsCsv.setEnabled(true);
 						closeText.setEnabled(true);
+            openConcordance.setEnabled(true);
```

Comment:
Do not mix tabs with spaces for indentation. Use the same indentation character thought the code.

![tab_spaces](https://cloud.githubusercontent.com/assets/800627/25286054/ac7b44a4-26bc-11e7-82c9-80ba2501da5b.png)